luigi/contrib/s3.py

```diff
@@ -486,7 +486,7 @@ def _get_s3_config(self, key=None):
         defaults = dict(configuration.get_config().defaults())
         try:
             config = dict(configuration.get_config().items('s3'))
-        except NoSectionError:
+        except (NoSectionError, KeyError):
```

Comment:
Am i correct to assume that ConfigParser returns the `NoSectionError` and toml returns the `KeyError` when a config section heading is missing?

Reply:
Yes. I think, this is incorrect raise from toml parser any exceptions from ConfigParser. Maybe, we can make custom common exception:

```python
class SectionError(NoSectionError, KeyError):
    pass
```

Reply:
I'm fine with it as it is currently written; I was just verifying that they raised different errors for the same problem.